.github/labeler.yml

```
@@ -1,3 +1,9 @@
-# Add needs triage label to all new issues
-needs-triage:
+# Add a needs triage for each browser to all new issues
+"needs-triage%3A chrome":
```

Since we're in quotes here, I don't think we need to encode the `:`? I did a quick test in https://github.com/oliverdunk/triage-test and it seems like that's the case.

---

Changes suggested by Oliver.

```suggestion
"needs-triage: chrome":
  - ""

"needs-triage: safari": 
  - ""

"needs-triage: firefox": 
```